core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -387,7 +387,12 @@ func (o *evmTxStore) preloadTxAttempts(txs []Tx) error {
 	return nil
 }
 
-func (o *evmTxStore) PreloadTxes(attempts []TxAttempt, qopts ...pg.QOpt) error {
+func (o *evmTxStore) PreloadTxes(attempts []TxAttempt) error {
+	return o.PreloadTxesAtomic(attempts)
+}
+
+// Only to be used directly for atomic transactions internal to the tx store
+func (o *evmTxStore) PreloadTxesAtomic(attempts []TxAttempt, qopts ...pg.QOpt) error {
```

Comment:
If this is internal, please rename to preloadTxesAtomic. No need to export this function